mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -33,6 +33,7 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
     private val stateStore: MvRxStateStore<S> = RealMvRxStateStore(initialState)
 ) : ViewModel() {
     private val debugMode = if (MvRxTestOverrides.FORCE_DEBUG == null) debugMode else MvRxTestOverrides.FORCE_DEBUG
+    private val forceUserTestObserver = MvRxTestOverrides.FORCE_USE_TEST_OBSERVER
```

Comment:
looks like a spelling mistake `user` instead of `use`

Is it necessary to add a property instead of referencing `FORCE_USE_TEST_OBSERVER` directly?

Reply:
Hm, actually on second thought , maybe I should call this `DISABLE_LIFECYCLE_AWARE_OBSERVER` . 
About using directly flag I was just trying to fallow same code style as you guys have for `debugMode` 
WYT ?

Reply:
@kojadin you can use it directly inline and the new name sounds good 👍

Reply:
👍 Done